src/js/components/sharedComponents/header/Dropdown.jsx

```diff
@@ -16,88 +15,74 @@ const propTypes = {
     items: PropTypes.array.isRequired
 };
 
-export default class Dropdown extends React.Component {
-    constructor(props) {
-        super(props);
+const Dropdown = ({
+    label,
+    title,
+    items
+}) => {
+    const [expanded, setExpanded] = useState(false);
 
-        this.state = {
-            expanded: false
-        };
+    const clickedButton = () => {
```

Comment:
This can be `toggleMenu` and we can delete `expandMenu ` and `collapseMenu`

Reply:
Not sure if we can actually. my changes aren't related to this but if we can improve w/o effecting UX I can change. it appears we need this to allow for expand/collapse on mouse enter/leave:

![image](https://user-images.githubusercontent.com/12897813/105408622-32ad3e00-5bfd-11eb-9149-4856cb2eba70.png)